frontend/src/app/components/MobileDialog/index.js

```diff
@@ -276,7 +276,7 @@ export default class MobileDialog extends React.Component {
       dimension13: fromFeatured ? "Featured Experiment" : "Experiment Detail"
     });
 
-    return subscribeToBasket(recipient, source).then(response => {
+    return subscribeToBasket(recipient, source, basketMsgId).then(response => {
```

Comment:
The intent was to pass an optional param in to become the newsletter value and otherwise default to test pilot